components/BottomSheet/tests/unit/BottomSheetShapeThemerTests.swift

```diff
@@ -15,7 +15,6 @@
 import XCTest
 import MaterialComponents.MaterialBottomSheet
 import MaterialComponents.MaterialBottomSheet_ShapeThemer
-import MaterialComponents.MaterialShapeLibrary
```

Comment:
Isn't this import still being used for `MDCCornerTreatment.corner(withRadius: 3)`? Did you mean to switch to the "New" version?

Reply:
Relying on a non-required import was causing pod lib lint to fail.
It works as expected now.
We may need to switch to the New version in the future, but it don't appear to be required yet.

Xcode tests pass as expected as do cocoapods.

```
$ pod lib lint --subspec=BottomSheet MaterialComponents.podspec 
MaterialComponents passed validation.
```

Reply:
I believe we should be using `MaterialComponent.MaterialShapeLibraryNew`.  That's where Yarden put the source until we can remove the original (private) ones.